test/preprocessor.tests.ts

```diff
@@ -217,25 +217,6 @@ describe("Preprocessor", () => {
             ]);
         });
 
-        it("#if (Foo != true) && Bar //Foo", async () => {
```

Comment:
Why was this removed? Do you not believe the test is useful or do we no longer support comments on preprocessor lines?